composer.json

```diff
@@ -82,7 +95,14 @@
         "phpunit/phpunit": "9.5.20",
         "squizlabs/php_codesniffer": "^3.6",
         "swoole/ide-helper": "4.8.9",
-        "textalk/websocket": "1.5.7"
+        "textalk/websocket": "1.5.7",
+        "phpbench/phpbench": "^1.2",
+
+        "symfony/console": "^5.0",
+        "symfony/filesystem": "^5.0",
+        "symfony/finder": "^5.0",
+        "symfony/options-resolver": "^5.0",
+        "symfony/process": "^5.0"
```

Comment:
This is because `phpbench` has a transitive dependency on these as version `^4 || ^5 || ^6`, and picks v6 by default which requires PHP 8.1, so causes runtime errors as our docker image is PHP 8.0